flatsurf/geometry/homology.py

```diff
@@ -361,6 +377,64 @@ def coefficient(self, gen):
 
         return self.coefficients()[index]
 
+    @cached_method
+    def simplex_coefficients(self):
```

Comment:
I think that
```
sage: (a - b)._chain.monomial_coefficients()
{(0, 1): 1, (0, 0): -1}
```
does the job already.

Reply:
Ok. I'll then just add this as an example to `chain()`.